dev-tools/scripts/smokeTestRelease.py

```diff
@@ -1376,7 +1144,7 @@ def main():
   c = parse_config()
 
   scriptVersion = scriptutil.find_current_version()
-  if not c.version.startswith(scriptVersion + '.'):
+  if not c.version.startswith(scriptVersion + '.') and c.version != scriptVersion:
```

Comment:
Could this be just this?

```suggestion
  if c.version != scriptVersion:
```